videoinput.c

```diff
@@ -492,6 +558,9 @@ void cVideoInput::Close()
       cCamSlot *cs = m_Device->CamSlot();
       if (cs)
         cs->CancelActivation();
+      if (m_Receiver)
```

Comment:
I have created a new setting for this strategy. Unfortunately it does not work reliable because when the device blacklists the channel it also unassigns the cam. As a result cs is null. So it only works for camSlot 0.
We need to store the camslot number on open. Working on it and push in an hour or so for you to review.